src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -99,6 +102,8 @@
     "@types/lodash": "4.14.161",
     "@types/node": "18.17.12",
     "@types/swagger-ui": "3.52.0",
+    "@types/vkbeautify": "0.99.4",
```

Comment:
What is this used for ?

Reply:
Not being used anymore. Removed it.
It was being used to prettify xml.